tests/test_models/test_necks.py

```diff
@@ -246,3 +246,34 @@ def test_dilated_encoder():
     feat = [torch.rand(1, in_channels, 34, 34)]
     out_feat = dilated_encoder(feat)[0]
     assert out_feat.shape == (1, out_channels, out_shape, out_shape)
+
+
+def test_ct_resnet_neck():
+    # num_filters/num_kernels must be a list
+    with pytest.raises(TypeError):
+        CTResNetNeck(
+            in_channel=10, num_deconv_filters=10, num_deconv_kernels=4)
+
+    # num_filters/num_kernels must be same length
+    with pytest.raises(AssertionError):
+        CTResNetNeck(
+            in_channel=10,
+            num_deconv_filters=(10, 10),
+            num_deconv_kernels=(4, ))
+
+    in_channels = 16
+    num_filters = (8, 8)
+    num_kernels = (4, 4)
+    ct_resnet_neck = CTResNetNeck(
+        in_channel=in_channels,
+        num_deconv_filters=num_filters,
+        num_deconv_kernels=num_kernels)
+    if torch.cuda.is_available():
```

Comment:
We may test the version that do not use DCN on CPU, and test the DCN version on GPU.